server/test/com/xsn/explorer/gcs/GolombEncodingSpec.scala

```diff
@@ -103,5 +136,100 @@ func main() {
       val expected = "056ff79e6c2994ba5d91402f327f807097c5c571f8d212511a8237f005331346102b41967f35ef488406c38a88"
       encoded.hex.string must be(expected)
     }
+
+    "return the correct filter for block at height 0 in the bitcoin testnet" in {
```

Comment:
You can actually import the json test file from the BIP and load it as a CSV which would avoid too much boilerplate for defining the test data, the blocks could be loaded with the `BlockLoader` and the transactions with `TransactionLoader` which loads a block/transaction from json files on the test resources, if you rebase against the develop, you'll find there is a way to split blocks by currency, we can grab the full json and place them in the btc folder (which you'll need to create) and use those helpers to avoid the boilerplate.